script/isotovideo

```diff
@@ -0,0 +1,201 @@
+#!/usr/bin/perl -w
+# Copyright 2009-2013 Bernhard M. Wiedemann
+# Copyright 2012-2020 SUSE LLC
+# SPDX-License-Identifier: GPL-2.0-or-later
+#
+
+=head1 SYNOPSIS
+
+isotovideo [OPTIONS] [TEST PARAMETER]
+
+Parses command line parameters, vars.json and tests the given assets/ISOs.
+
+=head1 OPTIONS
+
+=over 4
+
+=item B<-d, --debug>
+
+Enable direct output to STDERR instead of autoinst-log.txt
+
+=item B<--workdir=>
+
+isotovideo will chdir to that directory on startup
+
+=item B<--color=[yes|no]>
+
+Enable or disable color output explicitly. Defaults to "yes". Alternatively
+ANSI_COLORS_DISABLED or NO_COLOR can be set to any value to disable colors.
+
+=item B<-v, --version>
+
+Show the current program version and test API version
+
+=item B<-e, -?, --exit-status-from-test-results>
+
+isotovideo will exit with status 1 when a test module fails.
+
+=item B<-h, -?, --help>
+
+Show this help.
+
+=back
+
+=head1 TEST PARAMETERS
+
+All additional command line arguments specified in the C<key=value> format are
+parsed as test parameters which take precedence over the settings in the
+vars.json file. Lower case key names are transformed into upper case
+automatically for convenience.
+
+=head1 EXIT CODES
+
+=over 4
+
+=item B<0 - SUCCESS>
+
+isotovideo successfully executed a complete run. In the case of C<--exit-status-from-test-results> flag additionally all test modules passed or softfailed.
+
+=item B<1 - ERROR>
+
+An error ocurred during the test execution related to a test backend failure.
+
+=item B<100 - NO TEST MODULES SCHEDULED>
+
+No test module was scheduled.
+
+This exit code can only be reported when invoked with the
+C<--exit-status-from-test-results> flag.
+
+=item B<101 - TEST MODULES FAILED>
+
+At least one test module did not end with result "ok" or "softfail".
+
+This exit code can only be reported when invoked with the
+C<--exit-status-from-test-results> flag.
+
+=back
+
+=cut
+
+use Mojo::Base -strict, -signatures;
+use autodie ':all';
+no autodie 'kill';
+
+use constant {
+    EXIT_STATUS_OK => 0,
+    EXIT_STATUS_ERR => 1,
+};
+
+# Avoid "Subroutine JSON::PP::Boolean::(0+ redefined" warnings
+# Details: https://progress.opensuse.org/issues/90371
+use JSON::PP;
+
+my $installprefix;    # $bmwqemu::topdir
+
+BEGIN {
+    # the following line is modified during make install
+    $installprefix = undef;
+
+    my ($wd) = $0 =~ m-(.*)/-;
```

Comment:
```suggestion
    use FindBin '$RealBin';
    use Cwd qw(realpath);
    my ($wd) = realpath "$RealBin/..";
```

Reply:
I strongly suggest we keep any real content changes to the scripts outside this PR. I really only moved isotovideo to script/isotovideo here. I don't understand what `my ($wd) = $0 =~ m-(.*)/-;` is doing. Could you please create your own PR for that change, we apply that first and I will rebase my PR on top of your changes?

Reply:
yep

Reply:
#2518 2518

Reply:
You still need to add the `/..` in your PR though

Reply:
done